service/history/historyEngine.go

```diff
@@ -199,9 +199,16 @@ func (e *historyEngineImpl) registerDomainFailoverCallback() {
 		e.shard.GetShardID(),
 		e.shard.GetDomainCache().GetDomainNotificationVersion(),
 		func(prevDomain *cache.DomainCacheEntry, nextDomain *cache.DomainCacheEntry) {
+			domainFailoverNotificationVersion := nextDomain.GetFailoverNotificationVersion()
+			shardNotificationVersion := e.shard.GetDomainNotificationVersion()
+			domainActiveCluster := nextDomain.GetReplicationConfig().ActiveClusterName
+
+			e.logger.Infof("Domain Failover Event: Shard: %v, Domain: %v, ID: %v, Failover Notification Version: %v, Active Cluster: %v, Shard Domain Notification Version: %v\n",
```

Comment:
Can you convert these parameters into tags?

Reply:
this log will only appears once, per shard.
the domain id tagged logger is in each processor